libs-haskell/test-utils/DA/Test/Process.hs

```diff
@@ -0,0 +1,43 @@
+-- Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+module DA.Test.Process
+  ( ShouldSucceed(..)
+  , callProcessSilent
+  , callProcessSilentError
+  , callProcessForStdout
+  , callCommandQuiet
+  ) where
+
+import Control.Monad (unless,void)
+import System.IO.Extra (hPutStrLn,stderr)
+import System.Exit (ExitCode(ExitSuccess),exitFailure)
+import System.Process (CreateProcess,proc,shell,readCreateProcessWithExitCode)
+
+newtype ShouldSucceed = ShouldSucceed Bool
+
+callProcessSilent :: FilePath -> [String] -> IO ()
+callProcessSilent cmd args =
+  void $ run (ShouldSucceed True) (proc cmd args)
+
+callProcessSilentError :: FilePath -> [String] -> IO ()
+callProcessSilentError cmd args =
+  void $ run (ShouldSucceed False) (proc cmd args)
+
+callProcessForStdout :: FilePath -> [String] -> IO String
+callProcessForStdout cmd args =
+  run (ShouldSucceed True) (proc cmd args)
+
+callCommandQuiet :: String -> IO ()
```

Comment:
Let’s leave it like this for now but `callCommandQuiet` and `callProcessSilent` is rather confusing naming. Let’s use `Quiet` or `Silent` for both, I don’t have a preference.